.github/workflows/deploy-enterprise-test.yml

```diff
@@ -23,19 +23,21 @@ jobs:
             deploy_ref: refs/heads/develop
             job_files: >-
               job_spec/AUTORIFT.yml
-              job_spec/INSAR_GAMMA.yml
               job_spec/INSAR_GAMMA_TEST.yml
               job_spec/RTC_GAMMA.yml
               job_spec/INSAR_ISCE_TEST.yml
               job_spec/INSAR_ISCE_BURST.yml
               job_spec/WATER_MAP.yml
               job_spec/WATER_MAP_TEST.yml
               job_spec/WATER_MAP_EQ.yml
+              job_spec/RIVER_WIDTH.yml
```

Comment:
```suggestion
```

Reply:
I was going to keep this as a straight revert and re-implement this change as a separate PR into `develop`